ftl/python/builder.py

```diff
@@ -33,76 +33,106 @@ class Python(builder.JustApp):
     def __init__(self, ctx):
         self.descriptor_files = [_REQUIREMENTS_TXT]
         self.namespace = _PYTHON_NAMESPACE
+        self.ctx = ctx
         super(Python, self).__init__(ctx)
 
     def __enter__(self):
         """Override."""
         return self
 
-    def _generate_overrides(self):
+    def _generate_overrides(self, py_version='python2.7'):
         return metadata.Overrides(
-            creation_time=str(datetime.date.today()) + "T00:00:00Z")
-
-    def CreatePackageBase(self, base_image, cache):
+            creation_time=str(datetime.date.today()) + "T00:00:00Z",
+            env={
+                "VIRTUAL_ENV":
+                "/env",  # not working properly, not sure why
```

Comment:
What's not working about this?

Reply:
if I just add the VIRTUAL_ENV var, python is not finding the packages installed when I attempt to run the python app in /app.  Adding PYTHONUSERBASE adds the correct deps and then the app runs.  Also not sure if it is acceptable but the app must be launched with python3.6 if using python3.6 even when the env vars are set.

Reply:
How are you testing this? I think just the VIRTUAL_ENV part should work. Let's debug instead of using the other env var.